File: project_euler/Problem063/Problem063.cpp
#include <iostream>
#include <cmath>
using namespace std;

class Problem063 {
public:
    void solution() {
        int result = 0;
        for (int i = 1; i < 10; ++i) {
            result += (int) ((double) 1 / (1 - log10(i)));
        }
        cout << result << endl;
    }
};

File: project_euler/Problem063/Problem063.js
function Problem063() {
    let result = 0
    for (let i = 1; i < 10; ++i) {
        result += Math.floor(1 / (1 - Math.log10(i)))
    }
    console.log(result)
}

module.exports = {
    Problem063 : Problem063
};
